build/prod-webpack.config.js

```diff
@@ -21,11 +21,11 @@ module.exports = require('./buildConfig')(
     },
     themeEntries,
     paths,
-    [extractThemesPlugin, ModuleFederationPlugin],
-    true,
-    undefined,
-    undefined,
-    [
+    plugins: [extractThemesPlugin, ModuleFederationPlugin],
+    prod: true,
+    publicPath: undefined,
+    cssPrefix: undefined,
```

Comment:
you can remove these I think

Reply:
yeah right
i was in doubt if we had to make them clearly specified and visible

Reply:
can you make the suggestion and remove it?